### PR TITLE
Data extractor updates

### PR DESCRIPTION
**Pull request by ryanmelt**

### CODE REVIEW

File: openc3-cosmos-init/plugins/packages/openc3-cosmos-tool-dataextractor/src/tools/DataExtractor/DataExtractor.vue [hunk 1]
```diff
@@ -830,7 +844,8 @@ export default {
             } else {
               let rawVal = packet[key]['raw']
               if (Array.isArray(rawVal)) {
-                row[columnMap[key]] = '"[' + rawVal + ']"'
+                // row[columnMap[key]] = '"[' + rawVal + ']"'
+                row[columnMap[key]] = 'BINARY'
```
Comment (jmthomas) [inline openc3-cosmos-init/plugins/packages/openc3-cosmos-tool-dataextractor/src/tools/DataExtractor/DataExtractor.vue:848]: At a glance I thought this was outputting ARRAY items but they don't come across with 'raw' right? How would you extract raw binary?

File: openc3/lib/openc3/io/json_rpc.rb [hunk 2]
```diff
@@ -56,11 +56,15 @@ def as_json(options = nil) self end #:nodoc:
...
+  NON_UTF8_PRINTABLE = /[\x00-\x08\x0E-\x1F\x7F]/
   def as_json(options = nil)
     as_utf8 = self.dup.force_encoding('UTF-8')
     if as_utf8.valid_encoding?
-      return as_utf8
+      if as_utf8 =~ NON_UTF8_PRINTABLE
+        return self.to_json_raw_object
+      else
+        return as_utf8
+      end
```
Comment (jmthomas) [inline openc3/lib/openc3/io/json_rpc.rb:67]: Where did you run into this? Raw binary again?

File: openc3/lib/openc3/utilities/process_manager.rb [hunk 3]
```diff
@@ -52,7 +52,7 @@ def start
 
   # Spawns short lived processes and ensures they complete
   class ProcessManager
-    MONITOR_CYCLE_SECONDS = 10
+    MONITOR_CYCLE_SECONDS = 5
```
Comment (jmthomas) [inline openc3/lib/openc3/utilities/process_manager.rb:55]: 👍

**Review by jmthomas — APPROVED** [review]

File: openc3-cosmos-init/plugins/packages/openc3-cosmos-tool-dataextractor/src/tools/DataExtractor/DataExtractor.vue (hunk 1 above)
Comment (ryanmelt) [reply openc3-cosmos-init/plugins/packages/openc3-cosmos-tool-dataextractor/src/tools/DataExtractor/DataExtractor.vue:848]: Array items aren't raw.   I'll write a ticket to get the JSON as an output.

File: openc3/lib/openc3/io/json_rpc.rb (hunk 2 above)
Comment (ryanmelt) [reply openc3/lib/openc3/io/json_rpc.rb:67]: Raw binary with just small numbers is what we put into the INST IMAGE

**Pull request merged**